pkg/client/apitoken/client.go

```diff
@@ -0,0 +1,86 @@
+/*
+ * @license
+ * Copyright 2025 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package apitoken
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	coreapi "github.com/dynatrace/dynatrace-configuration-as-code-core/api"
+	corerest "github.com/dynatrace/dynatrace-configuration-as-code-core/api/rest"
+)
+
+const apiTokenPath = "/api/v2/apiTokens"
+
+type Response struct {
```

Comment:
The spec `.../rest/v2/rest-api-docs/v2/spec3.json` doesn't mention which fields are required, but it should be fine if the defaults are OK 👍

Reply:
I just created a token and validated the fields that are given. If something is not required it won't hurt us, as it will then be the empty value of the type (we don't have a strict validation like in account-management)